OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -203,27 +203,24 @@ public virtual void Tick()
 			if (!Active)
 				return;
 
-			if (Active)
-			{
-				var power = Instances.First();
-				if (Manager.DevMode.FastCharge && remainingSubTicks > 2500)
-					remainingSubTicks = 2500;
+			var power = Instances.First();
+			if (Manager.DevMode.FastCharge && remainingSubTicks > 2500)
+				remainingSubTicks = 2500;
 
-				if (remainingSubTicks > 0)
-					remainingSubTicks = (remainingSubTicks - 100).Clamp(0, TotalTicks * 100);
+			if (remainingSubTicks > 0)
+				remainingSubTicks = (remainingSubTicks - 100).Clamp(0, TotalTicks * 100);
 
-				if (!notifiedCharging)
-				{
-					power.Charging(power.Self, Key);
-					notifiedCharging = true;
-				}
+			if (!notifiedCharging)
+			{
+				power.Charging(power.Self, Key);
+				notifiedCharging = true;
+			}
 
-				if (RemainingTicks == 0
-					&& !notifiedReady)
-				{
-					power.Charged(power.Self, Key);
-					notifiedReady = true;
-				}
+			if (RemainingTicks == 0
+				&& !notifiedReady)
```

Comment:
Style nit: Can we move this on the above line while we're here?

Reply:
Done.